File: Frontend/src/app/DTOs/Group/GroupOverviewDTO.ts
﻿export interface GroupOverviewDTO {
    groupId: string;
    groupName: string;
    profileImagePath: string;
    lastMessageSentByMe: boolean | null;
    lastMessage: string | null;
    lastMessageDateSent: Date | null;
    lastMessageWasInfoMessage: boolean | null;
    numberOfFiles: number;
}

File: Backend/DTOs/Group/GroupOverviewDTO.cs
namespace Learnz.DTOs;
public class GroupOverviewDTO
{
    public Guid GroupId { get; set; }
    public string GroupName { get; set; }
    public string ProfileImagePath { get; set; }
    public bool? LastMessageSentByMe { get; set; }
    public string? LastMessage { get; set; }
    public DateTime? LastMessageDateSent { get; set; }
    public string? LastMessageSentUsername { get; set; }
    public bool? LastMessageWasInfoMessage { get; set; }
    public int NumberOfFiles { get; set; }
}
